crates/bevy_ecs/src/schedule/schedule.rs

```diff
@@ -1710,6 +1732,22 @@ impl ScheduleBuildSettings {
     }
 }
 
+/// list of [`ComponentId`]'s to ignore when reporting ambiguity conflicts between systems
+#[derive(Resource, Default, Clone)]
+pub struct IgnoreSchedulingAmbiguities(Vec<ComponentId>);
+
+impl IgnoreSchedulingAmbiguities {
+    /// Ignore ambiguities between systems in [`Component`] T.
```

Comment:
```suggestion
    /// Ignore ambiguities between systems on [`Component`] T.
```

Reply:
There's a couple more occurences of these.

Reply:
Is "with" a better word here? "on" still feels a little awkward.

Reply:
So, it doesn't quite work like "with" or "of": they can still have conflicts on other components, even if they both use T.

```suggestion
    /// Ignore system order ambiguities caused by conflicts on [`Component`]s of type `T`.
```

Reply:
this is much more clear. Thanks.